environment.py

```diff
@@ -0,0 +1,65 @@
+from typing import Optional
+import numpy as np
+import gymnasium as gym
+from gymnasium.spaces import OneOf, Box, Dict
+import client
+import cv2
+
+
+class TankwarsEnv(gym.Env):
+
+    def __init__(self, client):
+        self.client = client
+        self.observation_space = Box(0, 255, shape=(200, 200, 3), dtype=np.uint8)
+        self.action_space = Dict({
+            "right_engine": Box(-1, 1, shape=(1,), dtype=np.float32),
+            "left_engine": Box(-1, 1, shape=(1,), dtype=np.float32),
+            "fire": OneOf(True, False),
+        })
+
+        def get_state(self, player_id):
+            self.client.request_observation(player_id, self.client.ObservationKind.IMAGE)
+            return self.client.player_state(player_id)
+
+        def get_info(self):
+            pass
+
+        def reset(self, seed=None):
+            # super().reset(seed=seed)
```

Comment:
Why is this commented? Is it intentional?

Reply:
It just recalls the __init__. I'm not sure if it's necessary or not because everything about the env is done in another part of the code.

Reply:
Its better to call `super().reset()` anyway, purely by convention